pkg/operation/botanist/controlplane.go

```diff
@@ -1132,6 +1133,12 @@ func (b *Botanist) DeployKubeAPIServer(ctx context.Context) error {
 	defaultValues["serviceAccountConfig"] = serviceAccountConfigVals
 	defaultValues["admissionPlugins"] = admissionPlugins
 
+	if mountHostCADirectories {
```

Comment:
Nit: there is no need from this if statement, right? I guess, we could directly have
```suggestion
		defaultValues["mountHostCADirectories"] = map[string]interface{}{
			"enabled": mountHostCADirectories,
		}
```